packages/charts/src/charts/table/table-chart.component.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import HeaderRow from './header-row.component';
+import Row from './row.component';
+import { TableContainer, StyledTable } from './table.styles';
+import { sortData } from './table.utils';
+import { FormatFuncType } from '../../types';
+import { firstCapital } from './table.utils';
+
+export type Props = {
+  /** Chart data */
+  data: any[];
+  /** Main color */
+  color: string;
```

Comment:
I believe the `plot` should accept `theme` interface - so we could have unified `Props` for most of visualization components.